src/server/config.rs

```diff
@@ -142,6 +144,7 @@ impl Default for Config {
             end_point_request_max_handle_duration: ReadableDuration::secs(
                 DEFAULT_ENDPOINT_REQUEST_MAX_HANDLE_SECS,
             ),
+            end_point_max_concurrency: cpu_num,
```

Comment:
What is this config used for?

Reply:
It is not a config for yatp. It just limits concurrent running coprocessor requests to prevent starving and OOM.

Reply:
Hmm. But why do we need to limit concurrent tasks? What will happen if we have 8 cores and there are 8 big requests? Will small requests being blocked?

Reply:
Tt's truely a problem I'm aware of, but a bit difficult to solve now.

Reply:
But if we don't limit concurrent tasks, older big requests can always be in the low priority queue, but we are always open to new requests. That's a risk of starving and OOM.